configure.ac

```diff
@@ -454,7 +454,6 @@ AC_CHECK_TYPES([socklen_t], [], [], [
 dnl These are defined elsewhere than stdio.h.
 PHP_CHECK_SIZEOF([intmax_t], [0])
```

Comment:
Potential follow-up, we don't need to check for `intmax_t` as it is a C99 type and we require C99